bridges/primitives/chain-bridge-hub-cumulus/src/lib.rs

```diff
@@ -54,6 +54,14 @@ const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(constants::WEIGHT_REF_TI
 	.saturating_div(2)
 	.set_proof_size(polkadot_primitives::MAX_POV_SIZE as u64);
 
+/// We allow for 2 seconds of compute with a 6 second average block.
+///
+/// This is a copy-paste of system parachain runtimes switched to async backing (https://github.com/paritytech/polkadot-sdk/pull/2949).
+const MAXIMUM_BLOCK_WEIGHT_FOR_ASYNC_BACKING: Weight = Weight::from_parts(
```

Comment:
This makes no sense? Either the runtime is running using async backing or not. This means you should update `BlockWeights` directly.

Reply:
Runtime's `BlockWeights` are updated directly in that https://github.com/paritytech/polkadot-sdk/pull/2949.

Here we try to keep bridges primitives in sync.
Bridges repo is [partially migrated](https://github.com/paritytech/parity-bridges-common/issues/2538) to the `polkadot-sdk`, the relayer part is missing, and we use these primitives for relayer code also (I will double-check if it is really needed now), that's the reason we have these "duplicite" copies. From time to time, we sync/backport bridges repo with polkadot-sdk.

Reply:
Hmm okay.